### PR TITLE
Download large data files before build

### PR DESCRIPTION
**Pull request by kozo2**

### CODE REVIEW

File: src/MSDIAL4/MsDial/MSDIAL.csproj [hunk 1]
```diff
@@ -37,7 +37,16 @@
...
     <PublishWizardCompleted>true</PublishWizardCompleted>
     <BootstrapperEnabled>true</BootstrapperEnabled>
+    <Lbm2Url>http://prime.psc.riken.jp/compms/msdial/download/lipidblast/LipidMsmsBinaryDB-VS68-AritaM.lbm2</Lbm2Url>
   </PropertyGroup>
+  <Target Name="DownloadContentFiles" BeforeTargets="Build">
+		<DownloadFile
+            SourceUrl="$(Lbm2Url)"
+            DestinationFolder="$(OutputPath)">
+			<Output TaskParameter="DownloadedFile" ItemName="Content"
+			Condition="Exists('$(OutputPath)\README.md')" />
```
[the remark this answers is not in the document]
Comment (kozo2) [reply src/MSDIAL4/MsDial/MSDIAL.csproj]: I updated to follow the comment in https://github.com/systemsomicslab/MsdialWorkbench/commit/53e610aba31004e45dd66f05f1736ea6e13f4102

File: src/MSDIAL5/MsdialGuiApp/MsdialGuiApp.csproj [hunk 2]
```diff
@@ -9,7 +9,16 @@
 		<ApplicationIcon>Resources\msdial_icon.ico</ApplicationIcon>
 		<PlatformTarget>AnyCPU</PlatformTarget>
 		<Configurations>Debug;Release;Debug vendor unsupported;Release vendor unsupported</Configurations>
+		<Lbm2Url>http://prime.psc.riken.jp/compms/msdial/download/lipidblast/LipidMsmsBinaryDB-VS68-AritaM.lbm2</Lbm2Url>
```
[the remark this answers is not in the document]
Comment (kozo2) [reply src/MSDIAL5/MsdialGuiApp/MsdialGuiApp.csproj]: I updated to follow the comment in https://github.com/systemsomicslab/MsdialWorkbench/commit/53e610aba31004e45dd66f05f1736ea6e13f4102

File: src/MSDIAL5/MsdialGuiApp/MsdialGuiApp.csproj [hunk 3]
```diff
@@ -9,7 +9,16 @@
...
 	</PropertyGroup>
+	<Target Name="DownloadContentFiles" BeforeTargets="Build">
+		<DownloadFile
+            SourceUrl="$(Lbm2Url)"
+            DestinationFolder="$(OutputPath)">
+			<Output TaskParameter="DownloadedFile" ItemName="Content"
+			Condition="Exists('$(OutputPath)\README.md')" />
```
[the remark this answers is not in the document]
Comment (kozo2) [reply src/MSDIAL5/MsdialGuiApp/MsdialGuiApp.csproj]: I updated to follow the comment in https://github.com/systemsomicslab/MsdialWorkbench/commit/53e610aba31004e45dd66f05f1736ea6e13f4102

File: src/MSDIAL5/MsdialGuiApp/MsdialGuiApp.csproj [hunk 4]
```diff
@@ -9,7 +9,16 @@
...
 		<PlatformTarget>AnyCPU</PlatformTarget>
 		<Configurations>Debug;Release;Debug vendor unsupported;Release vendor unsupported</Configurations>
+		<Lbm2Url>http://prime.psc.riken.jp/compms/code/Msp20221205132019_conventional_converted_dev.lbm2</Lbm2Url>		
 	</PropertyGroup>
+	<Target Name="DownloadContentFiles" BeforeTargets="Build">
+		<DownloadFile
+            SourceUrl="$(Lbm2Url)"
+            DestinationFolder="$(OutputPath)">
+			<Output TaskParameter="DownloadedFile" ItemName="Content"
+			Condition="!Exists('$(OutputPath)\*.lvm*')" />
```
Comment (YukiMatsuzawa) [inline src/MSDIAL5/MsdialGuiApp/MsdialGuiApp.csproj:19]:
```suggestion
			Condition="!Exists('$(OutputPath)\*.lbm*')" />
```

**Review by YukiMatsuzawa — CHANGES_REQUESTED** [review]

**Pull request merged**